schedule/yast/maintenance/offline_migration_15sp4.yaml

```diff
@@ -0,0 +1,27 @@
+name: offline_migration_15sp4
```

Comment:
could you please merge this with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18993/files#diff-9d62bbe8ca457a3f2d8c355eb12e0ac1f646c91a22daf7ac3bc593ca86f912e0R1-R21
so we have single file, we can improve in the other one to have a description and also the `clone` module.

Reply:
> could you please merge this with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18993/files#diff-9d62bbe8ca457a3f2d8c355eb12e0ac1f646c91a22daf7ac3bc593ca86f912e0R1-R21 so we have single file, we can improve in the other one to have a description and also the `clone` module.

No problem, just one thing we shouldn't set NO_ADD_MAINT_TEST_REPOS=1 to prevent us from adding unreleased repos. We shouldn't set it on both two jobs.

You can see from [**VR**](https://openqa.suse.de/tests/14035584#step/patch_and_reboot/1), we don't add any MU repos here.

Reply:
> > could you please merge this with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18993/files#diff-9d62bbe8ca457a3f2d8c355eb12e0ac1f646c91a22daf7ac3bc593ca86f912e0R1-R21 so we have single file, we can improve in the other one to have a description and also the `clone` module.
> 
> No problem, just one thing we shouldn't set NO_ADD_MAINT_TEST_REPOS=1 in the 1st job.
> 
> You can see from [**VR**](https://openqa.suse.de/tests/14035584#step/patch_and_reboot/1), we don't show any MU repos here.

repos are added now during installation, https://openqa.suse.de/tests/14035584#step/patch_and_reboot/21
that settings seems fine, why would we add it twice?